src/styles/utils/_mixin.scss

```diff
@@ -0,0 +1,15 @@
+@mixin size {
+  width: 500px;
+  height: 500px;
+}
```

Comment:
mb make sense to create some reusable mixin where you can pass size and create square